docs/ldml/tr35-keyboards.md

```diff
@@ -834,13 +967,32 @@ The following elements are not imported from the source file:
 * `names`
 * `settings`
 
+-->
+
 * * *
 
-### 5.11 <a name="Element_displayMap" href="#Element_displayMap">Element: displayMap</a>
+### <a name="Element_displayMap" href="#Element_displayMap">Element: displayMap</a>
+
+The displayMap can be used to describe what is to be displayed on the keytops for various keys. For the most part, such explicit information is unnecessary since the `@to` element from the `keys/key` element can be used. But there are some characters, such as diacritics, that do not display well on their own and so explicit overrides for such characters can help.
+Another useful scenario is where there are doubled diacritics, or multiple characters with spacing issues.
+
+The `displayMap` consists of a list of display subelements.
+
+
+displayMaps are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` in where needed.
+
+For combining characters, U+25CC `◌` is used as a base.
```

Comment:
```suggestion
For combining characters, U+25CC `◌` is used as a base. It is an error to use a combining character without a base in the `display` attribute.
```

Reply:
Good suggestion. There may be an existing Unicode term for this.